x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/ilm/IndexLifecycleExplainResponse.java

```diff
@@ -221,7 +230,8 @@ private IndexLifecycleExplainResponse(
         String snapshotName,
         String shrinkIndexName,
         BytesReference stepInfo,
-        PhaseExecutionInfo phaseExecutionInfo
+        PhaseExecutionInfo phaseExecutionInfo,
+        Integer actionsOrderVersion
```

Comment:
We treat this as non-nullable elsewhere with the check in the state against null and setting it to `1`. Should we remove the nullification here and make it a regular `int`, enforcing it to be 1 if it's unset?